### DISCUSSION TITLE
The X11 connection broke (error 1).

### DISCUSSION THREAD

**Discussion by wowodejiajia** [idaholab/moose#19522] [Q&A General] [answered]
Dear MOOSE developers,

I ran it on Peacock for two days and then reported an error.  How can I solve it?  

(moose) phlab@DESKTOP-PRQA4JN:~/projects/donkey/problems$ peacock -i ex02.i
QStandardPaths: XDG_RUNTIME_DIR not set, defaulting to '/tmp/runtime-phlab'
Found executable: /home/phlab/projects/donkey/donkey-opt
Exodus filenames: ['peacock_run_exe_tmp_ex02_other.e']
Postprocessor filenames: ['peacock_run_exe_tmp_ex02_out.csv']
Running command: mpiexec -n 16 /home/phlab/projects/donkey/donkey-opt Outputs/csv=true -i peacock_run_exe_tmp_ex02.i
Working directory: /home/phlab/projects/donkey/problems
The X11 connection broke (error 1). Did the X11 server die?
![error](https://user-images.githubusercontent.com/85658962/144354545-b40e8805-f25b-4dd4-bc01-9811173284fd.png)

**Comment by GiudGiud** [comment, marked as answer]
Hello

hard to know. Maybe it did. 
You should try to recover from the checkpoint or exodus output, see this page: 
https://mooseframework.inl.gov/application_usage/restart_recover.html
and see if the simulation runs well again or if it dies (and that s what caused the broken connection?)

Guillaume

**Reply by wowodejiajia** [reply]
Thank you.
